lib/translator/report_luns.cc

```diff
@@ -91,17 +90,14 @@ StatusCode ReportLunsToScsi(const nvme::GenericQueueEntryCmd& identify_cmd,
     DebugLog("Namespace pointer was null");
     return StatusCode::kFailure;
   }
-
   uint32_t lun_count = GetNsListLength(*ns_list);
-  // TODO: TEMP TESTING.
-  // uint32_t lun_count = 1;
 
   // Define scsi response structure and determine attributes
   uint32_t allocated_list_bytes =
       buffer.size() - sizeof(scsi::ReportLunsParamData);
   scsi::ReportLunsParamData rlpd = {};
   rlpd.list_byte_length = htonl(lun_count * sizeof(scsi::LunAddress));
-  if (ntohl(rlpd.list_byte_length) > allocated_list_bytes) {
```

Comment:
we still need to call ntohl here because in the line above we made rlpd.list_byte_length big endian.